### PR TITLE
Create Topic_analysis.ipynb

### PR DESCRIPTION
**Pull request by jonathan-mcgraw**
Introductory topic analysis notebook

Contributors to add:
- Brogan Stich

## Description

An introductory notebook to show a propositioned method for topic analysis.

## Type of change

- [x] New feature

## Checklist:

- [x] My code follows PEP8 style guide
- [x] I have removed unnecessary print statements from my code
- [ ] I have made corresponding changes to the documentation if necessary
- [x] My changes generate no errors
- [x] No commented-out code
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

https://www.loom.com/share/a7a7f4c3e026450997de42ba66e97628

### CODE REVIEW

**Review by Zack-Quintana — APPROVED** [review]
Seems like a feature that will be useful in the future. It looks like you're already aware of the need to update the documentation so I don't have anything else to add. Nice

**Review by hollberg — APPROVED** [review]
Really nice artifact. Once Underdog Devs builds up a large volume of meeting notes, assessments, etc., being able to classify these records by topic will greatly simplify search and other semantic research on these documents.

**Review by mgrohnke — APPROVED** [review]
Great start to a future feature for this product.  Loom video explained everything well - good job!

**Review by jinjahninjah — APPROVED** [review]

File: notebooks/Topic_analysis.ipynb [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+    "from dataclasses import dataclass\n",
+    "import pandas as pd\n",
+    "import random\n",
+    "\n",
+    "# NLTK Imports\n",
+    "from nltk.tokenize import word_tokenize\n",
+    "from nltk.corpus import stopwords\n",
+    "from nltk.stem import WordNetLemmatizer\n",
+    "\n",
+    "# Gemsin Imports\n",
```
Comment (BakerJr1904) [inline notebooks/Topic_analysis.ipynb:19]: First time I've seen Gemsim used!! Good job!!

**Review by BakerJr1904 — APPROVED** [review]
This will be handy!

**Comment by jonathan-mcgraw** [comment]
> Great work everyone! The only feedback I have is the create_table() function is a little redundant with the random sampling from a hardcoded dict as this can result in duplicates; however, this is for data generation, not the actual topic analysis.
> 
> The only change I want is remove the boiler plate code from your PR description and it should be good to go 👍

Fixed it for random sentence generation.

**Review by jinjahninjah — COMMENTED** [review]
Like a boss! Great Job! That sentence generator will help in fine tuning your topic modeling down the line 👍

**Pull request merged**